integration_tests/index/search_utils.py

```diff
@@ -0,0 +1,43 @@
+import csv
```

Comment:
I think we're supposed to have our copyright message at the top of every file. Please and thank you.

When I'm next bored I might add it as another pre-commit hook [ODC issue referencehttps://github.com/opendatacube/datacube-core/issues/1310).

Reply:
Yes indeed, good catch.